zipkin-autoconfigure/storage-elasticsearch-http/src/main/java/zipkin/autoconfigure/storage/elasticsearch/http/ZipkinHttpElasticsearchStorageAutoConfiguration.java

```diff
@@ -25,21 +30,61 @@
 import zipkin.autoconfigure.storage.elasticsearch.ZipkinElasticsearchStorageAutoConfiguration;
 import zipkin.storage.StorageComponent;
 
+import static com.google.common.base.Preconditions.checkArgument;
+
 @Configuration
-@EnableConfigurationProperties(ZipkinElasticsearchHttpStorageProperties.class)
+@EnableConfigurationProperties(ZipkinHttpElasticsearchStorageProperties.class)
```

Comment:
I think this is more conventional this way? I changed it to mirror your change to the HTTP autoconfiguration.